bitnami/memcached/templates/deployment.yaml

```diff
@@ -46,6 +49,7 @@ spec:
                 secretKeyRef:
                   name: {{ template "memcached.fullname" . }}
                   key: memcached-password
+            {{- toYaml .Values.extraEnv | nindent 12 }}
```

Comment:
Could you apply this please?
```diff
+            {{- if .Values.extraEnv }}
             {{- toYaml .Values.extraEnv | nindent 12 }}
+            {{- end }}
```
The reason is that we are linting our YAMLs. If you run the command below, you'll find an error:

```console
$ helm template --values bitnami/memcached/values.yaml bitnami/memcached -x templates/deployment.yaml | yamllint -s -d "{extends: default, rules: {line-length: disable, trailing-spaces: disable, truthy: enable, document-start: disable, empty-lines: {max-end: 2} }}" -

  47:11     error    syntax error: could not find expected ':'
```

Why? Because the resulting YAML has the following aspect:

```yaml
...       
          env:
            - name: MEMCACHED_USERNAME
              value: ""
            - name: MEMCACHED_PASSWORD
              valueFrom:
                secretKeyRef:
                  name: release-name-memcached
                  key: memcached-password
            []
...
```

Despite it's a valid YAML to be run on K8s, we're trying to avoid this "ugly" YAMLs